### PR TITLE
[CRM app] updated links to go to specific app url

### PR DESCRIPTION
**Pull request by phpgeekfr**

### CODE REVIEW

File: libs/admin/src/lib/admin-panel/components/go-to/go-to.component.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<a mat-icon-button [matMenuTriggerFor]="homeMenu">
```
Comment (GrandSchtroumpf) [inline libs/admin/src/lib/admin-panel/components/go-to/go-to.component.html:1]: Use a button, it's not an anchor

File: libs/admin/src/lib/admin-panel/components/go-to/go-to.component.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  <mat-icon svgIcon="launch"></mat-icon><span matLine></span>
+</a>
+<mat-menu #homeMenu="matMenu">
+  <mat-list>
```
Comment (GrandSchtroumpf) [inline libs/admin/src/lib/admin-panel/components/go-to/go-to.component.html:5]: donc use mat-list in a mat-menu & flatten the structure
```html
<mat-menu #appMenu="matMenu">
  <button mat-menu-item [matMenuTriggerFor]="contentMenu">
    <mat-icon svgIcon="logo_archipel_content_fill"></mat-icon>
    <span>Archipel Content</span>
  </button>
  ...
</mat-menu>

<mat-menu #contentMenu="matMenu">
  <a mat-menu-item [href]="'catalog' | goToApp:'marketplace':type:id:suffix">
    <mat-icon svgIcon="marketplace"></mat-icon>
    <span>Marketplace</span>
  </a>
  ...
</mat-menu>

....
```

File: libs/admin/src/lib/admin-panel/components/go-to/go-to.component.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+a {
```
Comment (GrandSchtroumpf) [inline libs/admin/src/lib/admin-panel/components/go-to/go-to.component.scss:1]: Better: `a[mat-menu-item]`

File: libs/admin/src/lib/admin-panel/components/go-to/go-to.component.html (hunk 2 above)
Comment (GrandSchtroumpf) [reply libs/admin/src/lib/admin-panel/components/go-to/go-to.component.html:5]: Maybe if you can, use `*ngFor`

File: libs/admin/src/lib/admin-panel/components/go-to/go-to.component.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  templateUrl: './go-to.component.html',
+  styleUrls: ['./go-to.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class GoToComponent {
+  public appUrlMarket = appUrlMarket;
+  @Input() type: string = '';
+  @Input() id: string = '';
+  @Input() suffix: string = '';
+  @Input() festival = true;
```
Comment (GrandSchtroumpf) [inline libs/admin/src/lib/admin-panel/components/go-to/go-to.component.ts:15]: Maybe use an array instead.

File: libs/admin/src/lib/admin-panel/components/go-to/go-to.component.ts [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+@Component({
+  selector: 'admin-go-to',
+  templateUrl: './go-to.component.html',
+  styleUrls: ['./go-to.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class GoToComponent {
+  public appUrlMarket = appUrlMarket;
+  @Input() type: string = '';
+  @Input() id: string = '';
```
Comment (GrandSchtroumpf) [inline libs/admin/src/lib/admin-panel/components/go-to/go-to.component.ts:13]: Could you just giv the path instead ? 
```typescript
@Input() path: string;
```
It looks easier & more readable to me.

File: libs/admin/src/lib/admin-panel/pages/distribution-rights/distribution-rights.component.html [hunk 6]
```diff
@@ -13,7 +13,8 @@ <h1><a routerLink="/c/o/admin/panel/">Administration panel</a> >{{ movieId ? ' '
       <a target="_blank" [routerLink]="['/c/o/admin/panel/right',rightLink.id, 'm', rightLink.movieId]" routerLinkActive="active">{{ rightLink.id }}</a>
     </ng-template>
     <ng-template colRef="movieId" let-id>
-      <a target="_blank" [routerLink]="['/c/o/dashboard/tunnel/movie', id]" routerLinkActive="active"><mat-icon svgIcon="launch"></mat-icon></a>
+      <span>{{ id }} </span>
+      <admin-go-to type="movie" [id]="id"></admin-go-to>
```
Comment (GrandSchtroumpf) [inline libs/admin/src/lib/admin-panel/pages/distribution-rights/distribution-rights.component.html:17]: Shouldn't the type be "title" instead ?

File: libs/admin/src/lib/admin-panel/pipes/go-to.pipe.ts [hunk 7]
```diff
@@ -0,0 +1,35 @@
+import { Pipe, PipeTransform, NgModule } from '@angular/core';
+import { App, Module, applicationUrl } from '@blockframes/utils/apps';
+
+@Pipe({ name: 'goToApp', pure: true })
+export class GoToAppPipe implements PipeTransform {
+  transform(app: App, module: Module, type?: string, id?: string, suffix?: string) {
+    let path = '';
```
Comment (GrandSchtroumpf) [inline libs/admin/src/lib/admin-panel/pipes/go-to.pipe.ts:7]: It's up to you, but I think giving a path directly would be easier: 
```typescript
return `${applicationUrl[app]}/c/o/${module}${path}`
```

File: libs/admin/src/lib/admin-panel/components/go-to/go-to.component.ts (hunk 5 above)
Comment (phpgeekfr) [reply libs/admin/src/lib/admin-panel/components/go-to/go-to.component.ts:13]: This is what I wanted to do but the path to a given route can be different regarding the app (movie tunnel for example). So I decided to do the logic here

**Pull request merged**